gf-webapp-admin/src/component/sideMenu/addRoomModal/roomStartState.ts

```diff
@@ -16,4 +17,33 @@ export const defaultRoom = new NewRoom({
   rating: 0,
   services: [],
   square: 30,
+  roomNumber: 1,
+  booked: [
+    {
+      customer: {
+        id: "1",
+        name: "Victor",
+        email: "345@gmail.com",
+        phone: "+34343434",
+      },
+      period: {
+        dateIn: new Date(),
+        dateOut: new Date(),
+      },
+    },
+  ],
+  paid: [
+    {
+      customer: {
+        id: "1",
+        name: "Victor",
+        email: "345@gmail.com",
+        phone: "+34343434",
+      },
+      period: {
+        dateIn: new Date(),
+        dateOut: new Date(),
+      },
+    },
+  ],
```

Comment:
errr, why default room consist of some business data? It looks like stub. Imagine u are trying to create a new room. And It's already at the start includes weird paid, booked, data, Room.Single etc. It's strange a little)

Reply:
Create new issue #122